client/src/listings/CreateListing.js

```diff
@@ -69,39 +80,179 @@ export default function CreateListing() {
           </Grid>
         </Grid>
         <Divider />
-        <Grid container justify='space-between'>
-          <Grid item xs={5}>
+        <Grid container justify='space-between' spacing={2}>
+          <Grid item xs={6}>
             <TextField name='title' required fullWidth label='Title' autoFocus onChange={handleInputChange} />
           </Grid>
           {sessionStorage.getItem('accountType') === '1' ? (
             <Grid item xs={5}>
-              <FormControl fullWidth required onChange={handleInputChange}>
-                <InputLabel htmlFor='categoryId'>Category</InputLabel>
-                <Select name='categoryId' native defaultValue='' id='categoryId'>
+            <FormControl fullWidth required onChange={handleInputChange}>
+              <InputLabel htmlFor='categoryId'>Category</InputLabel>
+              <Select name='categoryId' native defaultValue='' id='categoryId'>
+                <option value='' />
+                {categoryOptions.length > 0 && Object.keys(subcategoryOptions).length > 0 ? (
+                  categoryOptions.map((cat) => (
+                    <optgroup key={cat.id} label={`${cat.name}`}>
+                      {subcategoryOptions[cat.id] ? (
+                        subcategoryOptions[cat.id].map((subcat) => (
+                          <option key={subcat.id} value={subcat.id}>{`${subcat.name}`}</option>
+                        ))
+                      ) : (
+                        <></>
+                      )}
+                    </optgroup>
+                  ))
+                ) : (
                   <option value='' />
-                  {categoryOptions.length > 0 && Object.keys(subcategoryOptions).length > 0 ? (
-                    categoryOptions.map((cat) => (
-                      <optgroup key={cat.id} label={`${cat.name}`}>
-                        {subcategoryOptions[cat.id] ? (
-                          subcategoryOptions[cat.id].map((subcat) => (
-                            <option key={subcat.id} value={subcat.id}>{`${subcat.name}`}</option>
-                          ))
-                        ) : (
-                          <></>
-                        )}
-                      </optgroup>
-                    ))
-                  ) : (
-                    <option value='' />
-                  )}
-                </Select>
-              </FormControl>
+                )}
+              </Select>
+            </FormControl>
+          </Grid>
+          ) : ( listing.categoryId='664' )}
```

Comment:
I think we can just get rid of this line, and move the logic to wherever it needs to go. You could even set this value when the page loads in a `useEffect` callback, if the account type is member.

We can even refactor this whole conditional here to just be (pseudocode):
```
// this:
{ isBusiness && categoryFieldJSX }
// instead of:
{ isBusiness ? categoryFieldJSX : empty }
```